app/routes/missions.fermi/route.mdx

```diff
@@ -36,6 +38,14 @@ import logo from './logo.svg'
 
 </div>
 
+<WithFeature GBM_JSON>
+**JSON-Serialized GCN Notices Types in GCN Kafka:**
+
+Fermi/GBM distributes JSON-serialized notices for the detection of gamma-ray transients on the GCN Kafka topic `gcn.notices.fermi.gbm`.
+
+For schemas and exmaples see [/schema/stable/gcn/notices/fermi/gbm](/schema/stable/gcn/notices/fermi/gbm) or use the [Schema Browser](/docs/schema/stable/gcn/notices/fermi/gbm/).
+</WithFeature>
```

Comment:
Please add following information with your modification, if any.
```suggestion

Detailed description and examples of Fermi/GBM Notices are available in the [GCN Schema GitHub project](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/fermi/gbm).

<div className="overflow-table">

| Type                                   | Contents                               | Latency |
| -------------------------------------- | -------------------------------------- | ------- |
| `gcn.notices.fermi.alert`                        | Localization, Count Rate, Significance | ~5 seconds  |

</div>

</WithFeature>
```
```suggestion
</WithFeature>
```

Reply:
@Vidushi-GitHub: Per @lpsinger, shouldn't we use  `gcn.notices.fermi.gbm`  for all notices?

Reply:
Corrected! please check and add as discussed.